tests/libc/tests.cmake

```diff
@@ -3,7 +3,7 @@
 
 string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE)
 string(TOUPPER ${CMAKE_C_COMPILER_ID} MY_COMPILER)
-set(BUILD "${MY_COMPILER}:${BUILD_TYYPE}")
+set(BUILD "${MY_COMPILER}:${BUILD_TYPE}")
```

Comment:
😅

Reply:
I noticed this and fixed it in my libc Windows enabling PR but that PR has been stalled for a while. So, good that it is getting fixed.